test/boost/mutation_reader_test.cc

```diff
@@ -1140,7 +1141,7 @@ SEASTAR_TEST_CASE(test_combined_reader_slicing_with_overlapping_range_tombstones
             readers.push_back(ds1.make_reader_v2(s, permit, query::full_partition_range, slice));
             readers.push_back(ds2.make_reader_v2(s, permit, query::full_partition_range, slice));
 
-            auto rd = downgrade_to_v1(make_combined_reader(s, permit, std::move(readers),
+            auto rd = mutation_fragment_v1_stream(make_combined_reader(s, permit, std::move(readers),
```

Comment:
Ditto for the rest of the tests, except those that are closely tied to a v1 (like the repair test).